internal/profile/_static/docker-compose-stack.yml

```diff
@@ -57,17 +57,19 @@ services:
       args:
         PROFILE: "${PROFILE_NAME}"
     healthcheck:
-      test: ["CMD", "curl", "--cacert", "/etc/ssl/package-registry/ca-cert.pem", "-f", "https://localhost:8080"]
+      test: ["CMD", "curl", "--cacert", "/etc/ssl/package-registry/ca-cert.pem", "-f", "https://localhost:8080/health"]
```

Comment:
I've been doing some quick tests, and this health endpoint fails (as I think requests to / would do) until the service has loaded all the packages:

```
$  curl -k -v https://localhost:8080/health
*   Trying 127.0.0.1:8080...
* Connected to localhost (127.0.0.1) port 8080 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (OUT), TLS handshake, Client hello (1):
* LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to localhost:8080
* Closing connection 0
curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to localhost:8080
````

The server starts running after all the packages have been read/validated and loaded. So IIRC if the service is live, then it is also ready, isn't it?

For now, I will revert this change and in this PR I will just set the ports and environment variables needed to enable/use Prometheus metrics.

Reply:
> Even if the package-registry (indexer) returns zero packages, it could be a valid scenario, couldn't it ? It would be the case where there are no packages to be served, but the service is running without errors
And if the indexer fails, IIRC the service exits.

Well, yes and no. We perform a dry-run validation to make sure that there are any packages, so I would say that we should expect from healthcheck a similar behavior.

> and this health endpoint fails (as requests to / do) until the service has loaded all the packages:

I think that the HTTP server starts after loading packages.